packages/react-ui/__tests__/widgets/TimeSeriesWidgetUI.test.js

```diff
@@ -233,4 +233,18 @@ describe('TimeSeriesWidgetUI', () => {
       jest.runAllTimers();
     });
   });
+
+  test('clear when animation is working', () => {
+    const onStop = jest.fn();
+    render(<Widget isPlaying={true} onStop={onStop} />);
+    fireEvent.click(screen.getByText(/Clear/));
+    setTimeout(() => expect(onStop).toHaveBeenCalledTimes(1), 250);
```

Comment:
Can't we avoid using a setTimeout for expect? Have you tried with async() and then screen.findByText(/Clear/)

Reply:
Inside testingjavascript materials, there is a little utility, called fire-event-async, that waits for next tick.
That could be useful here:

![image](https://user-images.githubusercontent.com/458196/146065523-194ad064-ac8c-4130-9b34-d67ee10d2c03.png)

PD: video from https://testingjavascript.com/lessons/svelte-use-dom-testing-library-with-svelte (01:26)

Reply:
So we can try to get a code with something like:

```
    const onStop = jest.fn();
    render(<Widget timeWindow={[1517785200000, 1528063200000]} onStop={onStop} />);
    fireEventAsync.click(await screen.findByText(/Clear/));
    expect(onStop).toHaveBeenCalledTimes(1);
```

Reply:
@VictorVelarde I've removed the `setTimeout `